plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/rating-filter.block_theme.spec.ts

```diff
@@ -38,7 +59,9 @@ test.describe( 'Product Filter: Rating Filter Block', async () => {
 
 			// wait for navigation
 			await page.waitForURL(
-				'/product-filters-rating-block/?rating_filter=1'
+				( url ) =>
+					url.searchParams.has( 'rating_filter' ) &&
```

Comment:
nit: This shouldn't be necessary since `get` returns `null` for unavailable params.

Reply:
Good point! Resolved in https://github.com/woocommerce/woocommerce/pull/44913/commits/001d009e7e38d650ee9a346171a05d2321e0448b